test/remote/gateways/remote_shift4_v2_test.rb

```diff
@@ -85,4 +85,25 @@ def test_failed_authorize
     assert_equal response.authorization, response.params['error']['chargeId']
     assert_equal response.message, 'The card was declined.'
   end
+
+  def test_successful_store_and_unstore
+    store = @gateway.store(@credit_card, @options)
+    assert_success store
+    assert_equal 'customer', store.params['objectType']
+    assert_match %r(^card_\w+$), store.params['cards'][0]['id']
+    assert card_id = store.params['defaultCardId']
+    assert customer_id = store.params['cards'][0]['customerId']
```

Comment:
Comment: Some asserts are not necessary because that ones was tested in https://github.com/activemerchant/active_merchant/blob/73cb68d6d5c6ef7041bd8a5397001d63cf2c11f5/test/remote/gateways/remote_securion_pay_test.rb#L37

Reply:
Hi @sinourain yes, since in this case the goal is the remove/unstore function and the store function already was implemented this extra check related to store are not needed, I will update it, thanks